packages/console/src/pages/Applications/components/GuideLibrary/index.tsx

```diff
@@ -70,7 +70,7 @@ function GuideLibrary({ className, hasCardBorder, hasCardButton }: Props) {
     setSelectedGuide(data);
   }, []);
 
-  const onCloseCreateForm = useCallback(
+  const onAppCreationFinalize = useCallback(
```

Comment:
nit: `Finalize` is a new naming phrase in our code base. Better avoid using new phrases?